File: copy-current-date.swift
#!/usr/bin/swift

// Required parameters:
// @raycast.schemaVersion 1
// @raycast.title Copy Current Date
// @raycast.mode silent
// @raycast.refreshTime 1d

// Optional parameters:
// @raycast.icon 🤖
// @raycast.packageName Current Date

// Documentation:
// @raycast.author Shawn Koh
// @raycast.authorURL https://shawnkoh.sg

import AppKit

let formatter = ISO8601DateFormatter()
formatter.formatOptions = .withFullDate
let date = formatter.string(from: Date())
NSPasteboard.general.clearContents()
NSPasteboard.general.writeObjects([date as NSPasteboardWriting])
print(date)

File: script-commands/copy-current-date.py
#!/usr/bin/env python3

# Required parameters:
# @raycast.schemaVersion 1
# @raycast.title Copy current date
# @raycast.mode silent
# @raycast.refreshTime 1h

# Optional parameters:
# @raycast.icon
# @raycast.packageName sg.shawnkoh.copy-current-date

# Documentation:
# @raycast.author Shawn Koh
# @raycast.authorURL https://github.com/shawnkoh

from datetime import *

date = datetime.today().strftime("%Y-%m-%d")
print(date)
